sentences/de/_common.yaml

```
@@ -122,6 +122,9 @@ expansion_rules:
   licht: "([das] Licht|[die] Lampe|[die] Beleuchtung)"
   lichter: "[die|der|von den] (Lichter|Lichtern|Lampen|Leuchten|Beleuchtungen)"
   alle_lichter: "(alle[r] <lichter>|von allen [Lichtern|Lampen|Leuchten|Beleuchtungen]|[die] ganze Beleuchtung)"
+  entsperren: "((ent|auf)(sperre|schlie(ss|ß)e)|öffne)[n]"
+  sperren: "(schlie(ss|ß)|sperr)[e|en]"
+  absperren: "(zu|ab)<sperren>"
```

```suggestion
  sperren: "(sperr|schlie(ss|ß))[e|en]"
  entsperren: "((ent|auf)<sperren>|öffne)[n]"
  absperren: "(zu|ab)<sperren>"
```
use the `<sperren>` expansion rule also for `entsperren`

---

I also thought about that but "entschließen" would just be wrong in that case ...?